bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+		<state>
+			<options>
+				<option value="PowerOff">Power Off</option>
+				<option value="Auto">Automatic</option>
+				<option value="Manual">Manual</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="ionizer">
+		<item-type>Switch</item-type>
+		<label>Ionizer Status</label>
+		<description>The ionizer status.</description>
+	</channel-type>
+
+	<channel-type id="uiLight">
+		<item-type>Switch</item-type>
+		<label>UI Light</label>
+		<description>The air quality light status.</description>
+	</channel-type>
+
+	<channel-type id="safetyLock">
+		<item-type>Switch</item-type>
+		<label>Safety Lock</label>
+		<description>The safety lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>Switch</item-type>
+		<label>Door Lock</label>
+		<description>The door lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorState">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="connectionState">
+		<item-type>Switch</item-type>
+		<label>Connection State</label>
+		<description>The connection state status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="startTime">
+		<item-type>Number:Time</item-type>
+		<label>Start Time</label>
+		<description>The start time of a washing.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="timeToEnd">
+		<item-type>Number:Time</item-type>
+		<label>Time To End</label>
+		<description>The time to end of a program.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="applianceUiSwVersion">
+		<item-type>String</item-type>
+		<label>Appliance UI SW Version</label>
+		<description>The appliance UI SW version.</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="applianceTotalWorkingTime">
+		<item-type>Number:Time</item-type>
+		<label>Appliance Total Working Time</label>
+		<description>The appliance total working time.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="optisenseResult">
+		<item-type>Number</item-type>
+		<label>Optisense Result</label>
+		<description>The optisense result.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="detergentExtradosage">
+		<item-type>Number</item-type>
+		<label>Detergent Extra Dosage</label>
+		<description>The detergent extra dosage.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="softenerExtradosage">
+		<item-type>Number</item-type>
+		<label>Softener Extra Dosage</label>
+		<description>The softener extra dosage.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="waterUsage">
+		<item-type>Number</item-type>
```

Comment:
Would it be possible to make this a UoM channel => Number:Volume ?

Reply:
Need to check that, I think it is in liter today.

Reply:
`"waterUsage":10,`

It is reported in litres, so we should be able to use `Number:Volume`.